pkgs/by-name/hu/hullcaster/package.nix

```diff
@@ -0,0 +1,31 @@
+{
+  lib,
+  fetchFromGitHub,
+  rustPlatform,
+}:
+
+rustPlatform.buildRustPackage rec {
+  pname = "hullcaster";
+  version = "v0.1.2";
+
+  src = fetchFromGitHub {
+    owner = "gilcu3";
+    repo = pname;
```

Comment:
Can you put the actual name of the repo here? Using `pname` in urls is discouraged because it can lead to annoying issues when overriding the name.
There's an issue about this somewhere, I think.

Reply:
https://github.com/NixOS/nixpkgs/issues/277994

Reply:
thanks, pushed an update